python/pyspark/sql/readwriter.py

```diff
@@ -415,28 +415,31 @@ def orc(self, path):
     def jdbc(self, url, table, column=None, lowerBound=None, upperBound=None, numPartitions=None,
              predicates=None, properties=None):
         """
-        Construct a :class:`DataFrame` representing the database table accessible
-        via JDBC URL `url` named `table` and connection `properties`.
```

Comment:
Single backticks in Sphinx just make things italic. e.g. See the [current doc](http://spark.apache.org/docs/1.6.1/api/python/pyspark.sql.html#pyspark.sql.DataFrameReader.jdbc).